### ISSUE TITLE
Функция загрузки датасета в память и разбиения на трейн/тест

### ISSUE THREAD

**Issue by GeekNekoS** [GeekNekoS/Chara#2]
Написать определение функции (def load_train_test...)
Аргументы:
directory: str - папка с категориями
batch_size: int - размер батча для обучения
image_size: tuple(int, int) - размер изображения
shuffle: bool = False - перемешивать или нет при формировании датасетов
seed: int = 42, семя для случайных процессов, для воспроизводимости результатов
validation_split: float = 0.8 - разделение на train/test
data_format: str = "channel_last" - формирование вектора, пример, если изображение rgb, то (224, 224, 3), где 3 это каналы синий, 
красный и зеленый
class_names: str = 'inferred'
Вывод:
train: tf.Dataset - тренировочный датасет
test: tf.Dataset - тестовый датасет

Для написания функции загрузки датасета в память и разбиения на трейн/тест использовать функцию image_dataset_from_directory из https://keras.io/api/data_loading/image/ для загрузки датасета и разбиения его на трейн/тест. Информация в помощь https://stackoverflow.com/questions/71129505/is-it-possible-to-split-a-tensorflow-dataset-into-train-validation-and-test-dat. Также в ней должна происходить аугментация данных на лету (если получится).
